packs/remedy/sensors/actions/delete.py

```diff
@@ -0,0 +1,7 @@
+from lib.actions import BaseAction
```

Comment:
This and following actions are under `packs/remedy/sensors`, is this omission, or intention?